back/engines/commercial/granular_permissions/config/routes.rb

```diff
@@ -6,7 +6,6 @@
       concern :permissionable do
         # We named the param :permission_action, bc :action is already taken (controller action).
         resources :permissions, param: :permission_action do
-          get 'participation_conditions', on: :member
```

Comment:
Still seems to be used by the front-end code. Check if actually used.

Reply:
I think @luucvanderzee is the right person to answer this indeed, I'm not enough up to date on the changes.